cmd/kubectl-gadget/traceloop.go

```diff
@@ -204,7 +122,23 @@ func runTraceloopStart(cmd *cobra.Command, args []string) error {
 }
 
 func runTraceloopStop(cmd *cobra.Command, args []string) error {
-	err := utils.DeleteTracesByGadgetName("traceloop")
+	traces, err := utils.ListTracesByGadgetName("traceloop")
+	if err != nil {
+		return err
+	}
+
+	if len(traces) == 0 {
+		return errors.New("please start traceloop before stopping it")
+	}
+
+	traceID := traces[0].Labels[utils.GlobalTraceID]
```

Comment:
We should handle the following cases:
* when `len(traces) > 1`.
* when `traceID == ""`

I could trigger the bug where `kubectl gadget traceloop stop` fails to stop traceloop with the following commands:
```
$ kubectl gadget traceloop start
$ kubectl apply -f  pkg/resources/samples/trace-traceloop.yaml 
trace.gadget.kinvolk.io/traceloop created
$ kubectl label -n gadget trace traceloop gadgetName=traceloop
trace.gadget.kinvolk.io/traceloop labeled
$ kubectl get trace -n gadget --show-labels
NAME              AGE   LABELS
traceloop         65s   gadgetName=traceloop
traceloop-wqsn9   98s   containerName=,gadgetName=traceloop,global-trace-id=gyocYopRBEOc17vI,namespace=,nodeName=,outputMode=Status,podName=,type=global
$ kubectl gadget traceloop stop
Error: failed to stop gadget: no traces found for traceID ""
$ kubectl gadget traceloop start
Error: traceloop was already started
```

And now we can neither stop or re-start traceloop.

This can be fixed in a future PR.